src/sql/workbench/test/browser/taskUtilities.test.ts

```diff
@@ -71,8 +74,9 @@ suite('TaskUtilities', function () {
 		mockConnectionManagementService.setup(x => x.isProfileConnected(TypeMoq.It.is(profile => profile === oeProfile || profile === tabProfile))).returns(() => true);
 
 		// Mock the workbench service to return the active tab connection
-		let tabConnectionUri = 'file://test_uri';
-		mockConnectionManagementService.setup(x => x.getConnectionProfile(tabConnectionUri)).returns(() => tabProfile);
+		const tabConnectionUri = URI.file('file://test_uri');
```

Comment:
I had to fix this test too - it was actually broken currently in a couple of ways : 

1. Creating the new ConnectionProfiles (oeProfile and tabProfile) weren't copying over the properties so they were essentially the same blank connection profiles
2. We never set up the activeEditor getter on the mock service so it doesn't actually do the logic the test is supposed to be testing (getting the profile from the current tab)

This was passing though because of 1, which meant that since both profiles were the same we were getting returned the oeProfile incorrectly, but it happened to have the same blank values as the tabProfile.